content/en/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md

```diff
@@ -149,6 +149,9 @@ resources:
   requests:
     memory: 128Mi
 ```
+{{< note >}}
+**Note**: Kubernetes rejects the  Container's memory request if it exceeds the default memory request specified by the LimitRange.
```

Comment:
i do not think this is accurate.

if a container makes no memory request, but a limit range has a default request, the container picks up the default request.

if a container has a memory request, the limit range should only evaluate that request relative to the min constraint.

Reply:
+1 to what @derekwaynecarr mentioned. This is inaccurate.